index.js

```diff
@@ -52,4 +52,168 @@ const state = {
     }
   ],
   cart: []
-};
\ No newline at end of file
+};
+
+
+//Access html elements
+const groceriesList = document.querySelector(".store--item-list")
+const cartContainer = document.querySelector(".cart--item-list-container")
+const cartItems = document.querySelector(".cart--item-list")
+const total = document.querySelector(".total-number")
+
+const products = state.items
+const productsInCart = state.cart
+// const cartItemsObj = {}
+// cartItemsObj.id = undefined
+// cartItemsObj.name = undefined
+// cartItemsObj.price = undefined
+// cartItemsObj.qtyInCart = 0
+
+
+
+//Create shop_list items
+function createShopLi() {
```

Comment:
I like that you created a function to create the different elements to use, makes it quite easy to read

Reply:
> I like that you created a function to create the different elements to use, makes it quite easy to read

The truth is that making it easy to read was not my goal. It just makes it easier for me to do the assembly  :D